charts/icm-as/values.yaml

```diff
@@ -92,6 +92,13 @@ nameOverride: ""
 
 fullnameOverride: ""
 
+# configure used ports
```

Comment:
Please do not make ports configurable via values.yaml - we only need them as references.

Reply:
But how to do it then? You asked for a `mgnt` interpolation something.

Reply:
> But how to do it then? You asked for a `mgnt` interpolation something.

Yes, if it's possible to reference the ICM-AS service ports via their service port declarations - if not, we don't need to change that. Unless @skiesewetter-intershop has some idea how to do that...

Reply:
Back to the old version